ccw.js

```diff
@@ -67,22 +67,15 @@ const prompt = () => {
             },
         ])
         .then((answers) => {
-            exec(`npm i ${project}`);
-
-            let raw;
-
-            try {
-                raw = readFileSync("package.json");
-            } catch (err) {
-                throw err;
-            }
+            execSync(`npm i ${project}`);
```

Comment:
`exec` is changed to `execSync` to make sure when `readFileSync` is called, the `package.json` file already has `cubecubed` as a dependency.